analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AsyncVoidMethod.CSharp9.cs

```diff
@@ -117,4 +117,17 @@ private async void EventHandlerCases_MyEvent(object sender, bool e)
             await Task.Run(() => Console.WriteLine("test"));
         }
     }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/5432
+    public class Reproducer5432
+    {
+        public void SomeMethod()
+        {
+            var _timer = new System.Threading.Timer(RunOnceAsync);
```

Comment:
Let's add also the `normal` overload with `TimerCallback callback` argument, to be sure we fix the `normal` case as well.

Reply:
Honestly I don't understand which overload you want to cover? This example is calling the `Timer` constructor with the `TimeCallback callback` as the argument. (`public Timer(TimerCallback callback)`)